.vscode/launch.json

```diff
@@ -0,0 +1,23 @@
+{
+	// Use IntelliSense to learn about possible attributes.
+	// Hover to view descriptions of existing attributes.
+	// For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+	"version": "0.2.0",
+	"configurations": [
+		{
+			"type": "node",
+			"request": "launch",
+			"name": "Launch Program",
+			"program": "${workspaceFolder}/test.js"
+		},
+		{
+			"type": "node",
+			"request": "launch",
+			"name": "Run AVA test",
+			"program": "${workspaceFolder}/node_modules/ava/profile.js",
+			"args": ["${workspaceFolder}/test.js",
+				  "--serial"],
+			"skipFiles": ["<node_internals>/**/*.js"]
+		}
+	]
+}
```

Comment:
This should not be part of this PR. I'm not interested in editor-specific config here.